reconcile/test/test_utils_ocm.py

```diff
@@ -41,3 +43,16 @@ def test_version_not_blocked_regex(self):
         self.ocm.blocked_versions = [r'^.*-fc\..*$']
         result = self.ocm.version_blocked('1.2.3-rc.1')
         self.assertFalse(result)
+
+
+class TestVersionRegex(TestCase):
+    @patch.object(OCM, '_init_access_token')
+    @patch.object(OCM, '_init_request_headers')
+    @patch.object(OCM, '_init_clusters')
+    # pylint: disable=arguments-differ
+    # pylint: disable=no-self-use
+    def test_invalid_regex(self, ocm_init_access_token,
+                           ocm_init_request_headers, ocm_init_clusters):
+        with pytest.raises(TypeError):
```

Comment:
why not using [`assertRaises`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises) directly and sticking to `unittest` here?

Reply:
just because i'm a noob.

Reply:
Yeah, as Rafa says,
```python
with self.assertRaises(TypeError):
    ...
```
is the unittest way.

Reply:
thx for fixing it.